frontend/src/scenes/surveys/SurveyEdit.tsx

```diff
@@ -53,51 +54,126 @@ import { SurveyFormAppearance } from './SurveyFormAppearance'
 import { ScheduleType, SurveyEditSection, surveyLogic } from './surveyLogic'
 import { surveysLogic } from './surveysLogic'
 
-export default function SurveyEdit(): JSX.Element {
-    const {
-        survey,
-        urlMatchTypeValidationError,
-        hasTargetingSet,
-        selectedPageIndex,
-        selectedSection,
-        isEditingSurvey,
-        targetingFlagFilters,
-        showSurveyRepeatSchedule,
-        schedule,
-        hasBranchingLogic,
-        surveyRepeatedActivationAvailable,
-        dataCollectionType,
-        surveyUsesLimit,
-        surveyUsesAdaptiveLimit,
-    } = useValues(surveyLogic)
-    const {
-        setSurveyValue,
-        resetTargeting,
-        resetSurveyResponseLimits,
-        resetSurveyAdaptiveSampling,
-        setSelectedPageIndex,
-        setSelectedSection,
-        setFlagPropertyErrors,
-        setSchedule,
-        deleteBranchingLogic,
-        setDataCollectionType,
-    } = useActions(surveyLogic)
-    const {
-        surveysMultipleQuestionsAvailable,
-        surveysRecurringScheduleAvailable,
-        surveysEventsAvailable,
-        surveysActionsAvailable,
-    } = useValues(surveysLogic)
-    const { featureFlags } = useValues(enabledFeaturesLogic)
-    const [visible, setVisible] = useState(false)
-    const sortedItemIds = survey.questions.map((_, idx) => idx.toString())
-    const { thankYouMessageDescriptionContentType = null } = survey.appearance ?? {}
+function SurveyRepeatSchedule(): JSX.Element {
+    const { showSurveyRepeatSchedule, survey, schedule } = useValues(surveyLogic)
+    const { setSurveyValue, setSchedule } = useActions(surveyLogic)
+    const { surveysRecurringScheduleAvailable } = useValues(surveysLogic)
+
     const surveysRecurringScheduleDisabledReason = surveysRecurringScheduleAvailable
         ? undefined
         : 'Upgrade your plan to use repeating surveys'
-    const surveysAdaptiveLimitsDisabledReason = surveysRecurringScheduleAvailable
-        ? undefined
-        : 'Upgrade your plan to use an adaptive limit on survey responses'
+
+    const canSurveyBeRepeated = survey.conditions?.events?.repeatedActivation
+
+    return (
+        <div className="mt-4">
+            <h3> How often should we show this survey? </h3>
+            {canSurveyBeRepeated ? (
+                <div className="flex flex-col font-medium text-muted">
+                    <span>
+                        <IconInfo className="mr-2" /> This survey is triggered by events:{' '}
+                    </span>
+                    <div>
+                        <LemonSnack className="mx-2">
+                            {survey.conditions?.events?.values.map((v) => v.name).join(', ')}
+                        </LemonSnack>
+                    </div>
+                    These settings don't apply since the survey will be shown each time these events occur.
+                </div>
```

Comment:
this is the only logic update

if the survey is set to be triggered after every time an event is captured, we show this message instead of showing the options to how many times an user can answer a survey

otherwise everything remains the same, I just split the big component into two smaller ones for easier readability